test/api-digester/stability-stdlib-abi.swift

```diff
@@ -1,4 +1,5 @@
 // REQUIRES: OS=macosx
+// REQUIRES: asserts
```

Comment:
This seems very dangerous. Can we have a with-asserts and a without-asserts variant, and say that the dump has to match one of them?

Reply:
It sounds like a good idea. I'll add another test requires no asserts

Reply:
It'd be easiest to keep them in sync if we assume the with-asserts variant is a superset. Then it can just use the `cat` of the two files as the expected changes.

Reply:
Just split the expected changes to two lists: one without asserts and the other the additional blames when asserts are enabled. The asserts-enabled test will diff against the combination of both.